speculos/mcu/vnc.py

```diff
@@ -60,15 +69,14 @@ def redraw(self, pixels):
         self.p.stdin.write(buf)
         self.p.stdin.flush()
 
-    def can_read(self, s, screen):
+    def can_read(self, s: int, screen: DisplayNotifier):
         '''Process a new keyboard or mouse event message from the VNC server.'''
 
-        assert s == self.s.fileno()
-        assert s == self.p.stdout.fileno()
+        assert s == self.fileno
 
         data = b''
         while len(data) != 6:
```

Comment:
should be interesting to have some comments here to describe all the numeric values used (description of the `data `array).

Reply:
It is an improvement for another PR. I have no idea what could lie inside `data` here, and structural changes do no change the current usage.